PopcornPals-Server/src/main/java/bg/fmi/popcornpals/controller/MediaController.java

```diff
@@ -0,0 +1,70 @@
+package bg.fmi.popcornpals.controller;
+
+
+import bg.fmi.popcornpals.dto.MediaDTO;
+import bg.fmi.popcornpals.mapper.MediaMapper;
+import bg.fmi.popcornpals.model.Media;
+import bg.fmi.popcornpals.service.MediaService;
+import bg.fmi.popcornpals.util.Genre;
+import bg.fmi.popcornpals.util.MediaType;
+import jakarta.validation.Valid;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/media")
```

Comment:
change endpoint to "api/media"

Reply:
i updated in acoiundint to this feed back in the next commit